test/old-start-configs/run.sh

```diff
@@ -7,9 +7,6 @@ OS_ROOT=$(dirname "${BASH_SOURCE}")/../../..
 source "${OS_ROOT}/hack/util.sh"
 cd "${OS_ROOT}"
 
-ensure_iptables_or_die
```

Comment:
> I expect this to be required so that the e2e scenario its running will work.

And this test **definitely** can't run against another server.  The whole point of this test to start a server up with  a specific config.